dev/release/02-source.sh

```diff
@@ -34,11 +38,17 @@ tagrc=${tag}-rc${rc}
 
 echo "Preparing source for tag ${tag}"
 
-: ${release_hash:=`git rev-list $tag 2> /dev/null | head -n 1 `}
+if [ ${PREPARE_RELEASE_TAG} -gt 0 ]; then
+  : ${release_hash:=`git rev-list $tag 2> /dev/null | head -n 1 `}
+else
+  : ${release_hash:=`git rev-parse HEAD `}
+fi
```

Comment:
We can already override `release_hash` by `release_hash=XXX dev/release/02-prepare.sh`. So we don't need this.

Reply:
I've set `release_hash` in `02-source-test.rb`.